### DISCUSSION TITLE
For Newcomers : Use my fork

### DISCUSSION THREAD

**Discussion by xAstroBoy** [winstxnhdw/lc-hax#525] [Show and tell]
im constantly adding new goodies to the fork, @winstxnhdw  is currently inactive , but im constantly adding new changes and new features to it.

https://github.com/xAstroBoy/lc-hax

**Comment by winstxnhdw** [comment]
Many thanks 🙏🏼

**Reply by arandomuser0** [reply]
@xAstroBoy fork is broken

**Reply by xAstroBoy** [reply]
> @xAstroBoy fork is broken

How if it works perfectly for me?

**Reply by arandomuser0** [reply]
you forgot to include the references

**Reply by xAstroBoy** [reply]
> you forgot to include the references

Fix the csproj path with the game's path.

**Reply by arandomuser0** [reply]
and its giving alot of compilation errors, not building the dll

**Reply by arandomuser0** [reply]
k

**Comment by arandomuser0** [comment]
> im constantly adding new goodies to the fork, @winstxnhdw is currently inactive , but im constantly adding new changes and new features to it.
> 
> https://github.com/xAstroBoy/lc-hax

still gave error

**Comment by arandomuser0** [comment]
even after fixing the path

**Comment by arandomuser0** [comment]
> im constantly adding new goodies to the fork, @winstxnhdw is currently inactive , but im constantly adding new changes and new features to it.
> 
> https://github.com/xAstroBoy/lc-hax

it gave me this error:
[ha2.txt](https://github.com/user-attachments/files/15827422/ha.txt)

**Reply by arandomuser0** [reply]
@xAstroBoy

**Reply by xAstroBoy** [reply]
refresh the nuget packages

**Reply by arandomuser0** [reply]
wdym how i do that @xAstroBoy

**Reply by arandomuser0** [reply]
@xAstroBoy can you just please send your local lc hax with it being a zip

**Comment by arandomuser0** [comment]
bro tell me @xAstroBoy

**Reply by arandomuser0** [reply]
how i fix it

**Comment by arandomuser0** [comment]
@xAstroBoy is just ghosting me like that? 😭

**Reply by winstxnhdw** [reply]
I’d appreciate if you move this discussion under his fork rather than here.

**Reply by xAstroBoy** [reply]
I investigated and fixed the issue.

**Comment by blueman5** [comment]
@xAstroBoy yo, i have a question, since you continued lc-hax i wonder if you please continue with https://github.com/BetaChess/LethalEscape  why? because this mod is too cool

**Reply by blueman5** [reply]
(add the new entities to it)

**Reply by blueman5** [reply]
@xAstroBoy

**Comment by blueman5** [comment]
or @xAstroBoy if you dont want that, make https://github.com/giosuel/imperium a clientside cheat

**Reply by blueman5** [reply]
(so you don't need to be host to use it) @xAstroBoy
